src/xcode/ENA/ENA/Source/Scenes/NewHome/HomeTableViewController.swift

```diff
@@ -87,6 +87,8 @@ class HomeTableViewController: UITableViewController, NavigationBarOpacityDelega
 		tableView.backgroundColor = .enaColor(for: .separator)
 
 		NotificationCenter.default.addObserver(self, selector: #selector(refreshUIAfterResumingFromBackground), name: UIApplication.didBecomeActiveNotification, object: nil)
+
+		viewModel.state.updateTestResult()
```

Comment:
The ViewModel is built after super.init and started here. Maybe you could use a method SetupViewModel to combine both here?

Reply:
Neither are setting up the view model. One place is binding the view to the view model and this on is triggering an update of the state, which is updating the view model. I don't see a relation between them.